Prevent Logstash from running as root. (#14046)
elastic/logstash: docs/static/settings-file.asciidoc

  and `NAME` is the name of the plugin.
| Platform-specific. See <<dir-layout>>.

| `on_superuser`

> | Flag to indicate properly utilizing superuser potentials when running Logstash. Offered options are `WARN`, `BLOCK` and `ALLOW`.

Reviewer: ```suggestion
| Setting to `WARN`, `BLOCK`, or `ALLOW` running Logstash as a superuser.
```

Author: Thanks @karenzone,
looks simple and clean, applied.
